index.html

```diff
@@ -17,23 +17,38 @@
     <div class="container">
       <br><br>
       <h1 class="header center grey-text lighten-5-text">Can I require on <a href="https://webtask.io" class="webtask-red-text" target="_blank">webtask.io</a>?</h1>
-      <div class="row hide-sticky">
-        <input type="text" class="input-main grey-text webtask-red-border" placeholder="https://webtask.myauth0.com/api/run/tenant" name="wt-url">
-        <input type="text" class="input-main grey-text webtask-red-border" placeholder="ey2312...." name="wt-token">
-      </div>
       <div class="row">
-        <div class="col s12 header-input-container" id="canirequire-search">
+        <div class="col s2 input-field">
+          <select id="version-dropdown">
+            <option value="8">Node 8</option>
+            <option value="4">Node 4</option>
+          </select>
+          <label>Node Runtime</label>
+        </div>
+        <div class="col s10 input-field" id="canirequire-search">
+          <label>Search for a module</label>
           <input type="text" class="input-main grey-text webtask-red-border" placeholder="e.g. mongodb" name="modules-filter">
         </div>
       </div>
+      <div class="col s12">
+        <div class="center">
+          Can't find the module you want? <a class="webtask-red-text" href="https://github.com/auth0-extensions/canirequire/issues/new">Request it</a>!
```

Comment:
We had taken this out of tehsis' version, probably because the requests were too many. Might want to consider removing here as well?

Reply:
Or...we could leave it and see what sort of requests we get.